apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/Logging/LogLevel.cs

```diff
@@ -15,9 +15,9 @@
 namespace Google.Cloud.Spanner.V1.Logging
 {
     /// <summary>
-    /// 
+    /// Log levels for Spanner client internal logging.
     /// </summary>
-    internal enum LogLevel
```

Comment:
ugh, I forgot about logging.  This was one of the more important ones to keep internal for when we switch to a more official logging/trace framework.
maybe we just move this to V1.Internal.Logging ns?